lib/ansible/modules/source_control/gitlab_group.py

```diff
@@ -5,23 +5,21 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: gitlab_group
 short_description: Creates/updates/deletes Gitlab Groups
 description:
    - When the group does not exist in Gitlab, it will be created.
-   - When the group does exists and state=absent, the group will be deleted.
+   - When the group does exist and state=absent, the group will be deleted.
```

Comment:
Add a note about Python library change like you did in gitlab_project PR. Rest LGTM

Reply:
Thanks! Added a line that I also have added in different PR for gitlab_project.